### PR TITLE
Created rent model. Created migration and migrated. Tested relationsh…

### PR DESCRIPTION
**Pull request by alanlamaswolox**
## Summary
Created the rent model
Created migrations and migrated
Tested the relationships

## Trello
https://trello.com/c/gMM5vjgM/21-13-crear-modelo-de-rent

### CODE REVIEW

File: app/models/rent.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class Rent < ApplicationRecord
+  validates :user_id, :book_id, :initial_date, :final_date, presence: true
+  belongs_to :user
```
Comment (nicozar95) [inline app/models/rent.rb]: In order to check for book presence you can use: `belongs_to :user, :book, required: true`

**Review by nicozar95 — CHANGES_REQUESTED** [review]
Good work, a really small change

File: app/models/user.rb [hunk 2]
```diff
@@ -10,4 +10,5 @@ class User < ApplicationRecord
   validates :first_name, presence: true
   validates :last_name, presence: true
   before_save :skip_confirmation!
+  has_one :rent, dependent: :destroy
```
Comment (nicozar95) [inline app/models/user.rb:13]: I think this is a has_many

File: db/migrate/20180919145724_create_rents.rb [hunk 3]
```diff
@@ -0,0 +1,15 @@
+class CreateRents < ActiveRecord::Migration[5.1]
+  def change
+    create_table :rents do |t|
+
+      #t.integer :user_id, null: false
```
Comment (nicozar95) [inline db/migrate/20180919145724_create_rents.rb]: Remove comments

File: spec/factories/rents.rb [hunk 4]
```diff
@@ -0,0 +1,4 @@
+FactoryBot.define do
```
Comment (nicozar95) [inline spec/factories/rents.rb]: Delete this factory if you are not using it

**Review by nicozar95 — CHANGES_REQUESTED** [review]
Last one

File: app/models/user.rb (hunk 2 above)
Comment (alanlamaswolox) [reply app/models/user.rb:13]: Ok, i didn't understand the domain model, i think that each user can has just only one rent at the same time. Ill change it now

**Review by nicozar95 — APPROVED** [review]
Short the boch!

**Pull request merged**